src/main/java/xyz/nucleoid/plasmid/widget/SidebarWidget.java

```diff
@@ -126,6 +128,14 @@ public Content writeLine(String line) {
             return this.writeRawLine(line);
         }
 
+        public Content writeFormattedTranslated(Formatting formatting, String key, Object args) {
```

Comment:
I imagine the args here should be varags?

Reply:
Just noticed that, whoops!